pkgs/development/libraries/linenoise/create-pkg-config-file.sh

```diff
@@ -0,0 +1,13 @@
+cat <<EOF > linenoise.pc
+prefix=$out
+exec_prefix=\${prefix}
+libdir=\${exec_prefix}/lib
+includedir=\${prefix}/include
+
+Name: linenoise
+Description: A minimal, zero-config, BSD licensed, readline replacement.
+Requires: 
+Version: 1.0.10
+Cflags: -I\${includedir}/ \${prefix}/src/linenoise.c
```

Comment:
Yes, the library will be compiled with the project that uses it. Yes, it is permitted. No, it's not nice.

I wanted to make a second pull request that provides at least a static library and at best a dynamically linked one. But I didn't even used that library since I added it to nixpkgs.